sorc/ncep_post.fd/MPI_FIRST.f

```diff
@@ -116,7 +119,18 @@ SUBROUTINE MPI_FIRST()
 !
 !     global loop ranges
 !
-      call para_range(1,jm,num_procs,me,jsta,jend)
+!      call para_range(1,jm,num_procs,me,jsta,jend)
+! GWVX temporary documentation
+!  para_range2 supports a 2D decomposition.  The rest of the post
+!  supports 1D still and the call here is the special case where each
+!  processor gets all of the longitudes in the latitude 1D subdomain
+!  jsta:jend.  The X decomposition will be specified by the third
+!  argument (currently 1) and the Y decoposition will be specified by
+!  the fourth argument (currently all of the ranks)   When X is
+!  subdivided the third and fourth arguments will have to be integral
+!  factors of num_procs and on 5/27/21 I am still working out a general
+!  way to do this if the user doesn't select the factors
+      call para_range2(im,jm,1,num_procs,me,ista,iend,jsta,jend)
```

Comment:
@GeorgeVandenberghe-NOAA this is good as a control case to match what UPP is doing now

Reply:
Yes.  It allows UPP to function in the 1D decomposition with the 2D decomposition support gradually inserted.  The big change and test will be when the third argument of para_range2 is larger than 1 which can't be done until all arrays  are dimensioned  with the  I subdomain.   Easy for MOST but I need to work out the halo stuff which both you and wen caught in the para_range2 evaluaton.